apps/cookbook/src/app/showcase/modal-experimental-showcase/modal-experimental-showcase.component.html

```diff
@@ -81,31 +91,48 @@ <h2 #fullscreenModal>Fullscreen modal</h2>
 
 <h2 #fullscreenModalSlots>Slots</h2>
 <p>
-  The modal component accepts 3 slots: <code>header-start</code>, <code>default</code> &
-  <code>footer</code>.
+  The modal component accepts 3 slots:
+  <code>header-start</code>
+  ,
+  <code>default</code>
+  &
+  <code>footer</code>
+  .
 </p>
 
 <h3>header-start</h3>
 
 <p>
-  The <code>header-start</code> slot places content in the upper left corner of the modal header. In
-  the example above a <code>kirby-page-progress</code> component is slotted in like this:
+  The
+  <code>header-start</code>
+  slot places content in the upper left corner of the modal header. In the example above a
+  <code>kirby-page-progress</code>
+  component is slotted in like this:
 </p>
 
 <cookbook-code-viewer [html]="headerStartSlotExampleTemplate"></cookbook-code-viewer>
 
 <h3>default</h3>
 
-<p>The <code>default</code> slot places the provided markup or component in the modals' content.</p>
+<p>
+  The
+  <code>default</code>
+  slot places the provided markup or component in the modals' content.
```

Comment:
Pardon my grammar, and i'm not sure, but I don't think the apostrophe should be on "modals"

Reply:
Nice catch! It should be "modal's", because I want to use the possesive form (I just looked that up 😂).